mozilla/foundation.mozilla.org
# added web monetization message to foundation site

Closes #7141 
Related PRs/issues #

**Link to sample test page**: https://foundation-s-7141-web-m-yafyg6.herokuapp.com/en/

**Steps to test:**

1. Visit the test environment linked above, and browse around
2. Note the addition of the new text at the bottom of the page footer
3. Link should link correctly and behave as expected
4. Visit the mozfest portion of the testing environment at https://mozfest-foundation-s-7141-web-m-yafyg6.mofostaging.net/en/
5. Scroll to the footer and note that the new text is NOT there.
6. If everything is behaving as expected, testing is complete!
